mvvmfx-testing-utils/src/main/java/de/saxsys/mvvmfx/testingutils/FxTestingUtils.java

```diff
@@ -2,10 +2,7 @@
 
 import javafx.application.Platform;
 
-import java.util.concurrent.CompletableFuture;
```

Comment:
Please revert the changed imports. I don't like star-imports as it makes it harder to see which classes are imported from where.